### PR TITLE
Hods 174 175 176

### PR DESCRIPTION
**Pull request by ajchilts**

### CODE REVIEW

**Review by owen-opencast — APPROVED** [review]

File: resources/public/api/conf/1.0/examples/number-of-employees-response.json [hunk 1]
```diff
@@ -4,31 +4,40 @@
...
-      "numberOfEmployees": 1666
-    },
-    {
-      "dateOfCount": "2020-02",
-      "numberOfEmployees": 1589
-    },
-    {
-      "dateOfCount": "2020-03",
-      "numberOfEmployees": 1555
+      "payeReference": "RT882d/456",
```
Comment (Luka-Abey) [inline resources/public/api/conf/1.0/examples/number-of-employees-response.json:9]: think there's been a duplication here of counts within counts/nested payeReference?

**Review by Luka-Abey — CHANGES_REQUESTED** [review]

**Review by Luka-Abey — APPROVED** [review]

File: test/unit/uk/gov/hmrc/organisationsdetailsapi/controllers/NumberOfEmployeesControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        """
+          |{
+          |    "_links": {
+          |        "self": {
+          |            "href": "/organisations/details/number-of-employees?matchId=32696d72-6216-475f-b213-ba76921cf459"
+          |        }
+          |    },
+          |    "employeeCounts": [
+          |        {
+          |            "payeReference": "RT882d/123",
```
Comment (Luka-Abey) [inline test/unit/uk/gov/hmrc/organisationsdetailsapi/controllers/NumberOfEmployeesControllerSpec.scala:131]:
```suggestion
          |            "payeReference": "123/RT882d",
```

File: test/unit/uk/gov/hmrc/organisationsdetailsapi/controllers/NumberOfEmployeesControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+      when(mockScopesService.getEndPointScopes("number-of-employees")).thenReturn(Seq("test-scope"))
+
+      when(mockAuthConnector.authorise(eqTo(Enrolment("test-scope")), refEq(Retrievals.allEnrolments))(any(), any()))
+        .thenReturn(Future.successful(Enrolments(Set(Enrolment("test-scope")))))
+
+      when(mockNumberOfEmployeesService.get(refEq(sampleMatchIdUUID), eqTo(sampleRequest),  eqTo(Set("test-scope")))(any(), any(), any()))
+        .thenReturn(Some(Seq(
+          NumberOfEmployeesResponse(
+            Some("RT882d/123"),
```
Comment (Luka-Abey) [inline test/unit/uk/gov/hmrc/organisationsdetailsapi/controllers/NumberOfEmployeesControllerSpec.scala:110]:
```suggestion
            Some("123/RT882d"),
```

**Review by Luka-Abey — CHANGES_REQUESTED** [review]
changed requested for consistent formation of payeReference

**Review by Luka-Abey — APPROVED** [review]

**Review by parkeradam — APPROVED** [review]

**Pull request merged**